src/renderer/api/controller.ts

```diff
@@ -223,6 +228,7 @@ const endpoints: ApiController = {
         scrobble: ssController.scrobble,
         search: ssController.search3,
         setRating: undefined,
+        shareItem: undefined,
```

Comment:
Strictly speaking (even though it wouldn't be used for now) Subsonic API also supports sharing https://subsonic.org/pages/api.jsp#createShare